cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,22 @@
+class ShippingStepPage {
+    private CheckTerm: string;
+    private CartNavigation: string
+
+    constructor() {
+        this.CheckTerm = '#cgv';
+        this.CartNavigation = '.cart_navigation span'
```

Comment:
This is referring to the proceed to checkout button, let's name this property in that way.